Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Generic/PlantEnable.mo

```diff
@@ -0,0 +1,395 @@
+  connect(con2.y, swi.u2)
+    annotation (Line(points={{-39,-180},{-20,-180},{-20,-120},{58,-120}},
+      color={255,0,255}));
+  connect(not2.y, mulOr.u1)
+    annotation (Line(points={{1,20},{20,20},{20,-12},{38,-12}},  color={255,0,255}));
+  connect(greEquThr2.y, mulOr.u2)
+    annotation (Line(points={{-39,-20},{38,-20}}, color={255,0,255}));
+  connect(hys1.y, swi.u2)
+    annotation (Line(points={{-39,-120},{58,-120}}, color={255,0,255}));
+  connect(wseTOut.TOutWet, TOutWet)
+    annotation (Line(points={{-142,-172},{-150,-172},{-150,-140},{-220,-140}},
+      color={0,0,127}));
+  connect(con3.y, wseTOut.VChiWat_flow)
+    annotation (Line(points={{-159,-180},{-142,-180}},
+      color={0,0,127}));
+  connect(wseTOut.uTunPar, uTunPar)
+    annotation (Line(points={{-142,-188},{-150,-188},{-150,-220},{-220,-220}},
+      color={0,0,127}));
+  connect(hasReq.y, not3.u)
+    annotation (Line(points={{-119,180},{-40,180},{-40,160},{-170,160},{-170,-20},
+      {-142,-20}}, color={255,0,255}));
+  connect(not3.y, enaTim1.u)
+    annotation (Line(points={{-119,-20},{-110,-20},{-110,-20},{-102,-20}}, color={255,0,255}));
+  connect(hys.y, not4.u)
+    annotation (Line(points={{-79,90},{-70,90},{-70,-60},{-62,-60}}, color={255,0,255}));
+  connect(not4.y, mulOr.u3)
+    annotation (Line(points={{-39,-60},{20,-60},{20,-28},{38,-28}}, color={255,0,255}));
+  connect(wseTOut.y, feedback.u2)
+    annotation (Line(points={{-118,-180},{-100,-180},{-100,-132}}, color={0,0,127}));
+  connect(TChiWatSupSet, feedback.u1)
+    annotation (Line(points={{-220,-100},{-140,-100},{-140,-120},{-112,-120}},
+      color={0,0,127}));
+
+annotation (
+  defaultComponentName = "plaEna",
+  Diagram(coordinateSystem(preserveAspectRatio=false, extent={{-200,-240},{200,240}})),
+  Icon(coordinateSystem(extent={{-100,-100},{100,100}}),
+       graphics={
+        Rectangle(
+          extent={{-100,100},{100,-100}},
+          lineColor={28,108,200},
+          fillColor={255,255,255},
+          fillPattern=FillPattern.Solid,
+          lineThickness=5,
+          borderPattern=BorderPattern.Raised),
+        Text(
+          extent={{-120,146},{100,108}},
+          lineColor={0,0,255},
+          textString="%name"),
+        Ellipse(extent={{-80,80},{80,-80}}, lineColor={28,108,200},fillColor={170,255,213},
+          fillPattern=FillPattern.Solid),
+        Ellipse(extent={{-90,90},{90,-90}}, lineColor={28,108,200}),
+        Rectangle(extent={{-75,2},{75,-2}}, lineColor={28,108,200},
+          fillColor={28,108,200},
+          fillPattern=FillPattern.Solid),
+        Text(
+          extent={{-66,46},{76,10}},
+          lineColor={28,108,200},
+          textString="START"),
+        Text(
+          extent={{-66,-8},{76,-44}},
+          lineColor={28,108,200},
+          textString="STOP")}),
+ Documentation(info="<html>
+<p>
+Block that generate chiller plant enable signals and output the initial plant stage,
```

Comment:
Maybe I should document WSE similarly, let me check in the next meeting and I can add it to this branch once this is ready.